### ISSUE TITLE
Forms & Validations + Accessibility Missing Prerequisites

### ISSUE THREAD

**Issue by StrawberryLente** [kamranahmedse/developer-roadmap#8944]
### Roadmap URL

https://roadmap.sh/frontend

### Suggestions

The sources for "Forms And Validations" under the HTML category require prior knowledge of JavaScript.
However, JavaScript is only taught after finishing HTML and CSS.
I suggest maybe moving the "Forms & Validations" segment to the JS category (or later).

**Comment by priyanshu-101** [comment]
Hi,
I have setup the codebase and will love to work on this issue
Can you please assign it to me

**Comment by StrawberryLente** [comment]
I've noticed the Accessibility sources also heavily rely on prior knowledge of CSS and JS, so this might also need to be reviewed.

**Comment by jcanalesluna** [comment]
Hi both! Thank you for your comments. We're constantly updating the roadmaps to improve the content. We will take care of your observations in the next iteration of the Frontend roadmap. Thanks

**Comment by nilbuild** [comment]
Thank you for reporting this, but this topic mainly refers to HTML's default validation features and not JavaScript.
